### ISSUE TITLE
Meaning of numbers in SH_Train.txt

### ISSUE THREAD

**Issue by joos2010kj** [tianyu0207/RTFM#66]
Hi,

Can I ask what the numbers in SH_Train.txt represent?

SH_Train dataset contains videos that are only formated in ##\_###.avi, but, in SH_Train.txt, I sometimes see numbers like 01_0014, 12_0175, and such, which are not formatted in ##\_###.  

Thanks!

**Comment by coranholmes** [comment]
Videos like 01_0014, 12_0175 are in the test set of the original ShanghaiTech dataset. The authors have mentioned in Section 4.1

> Zhong et al. [79] reorganised the data set by selecting a subset of anomalous testing videos into training data to build a weakly super- vised training set, so that both training and testing sets cover all 13 background scenes. We use exactly the same pro- cedure as in [79] to convert ShanghaiTech for the weakly supervised setting.
